### PR TITLE
Update abide2 and improve instruction

### PR DESCRIPTION
**Pull request by htwangtw**
@clarkenj Please see if this is sets of instruction makes more sense.
Combined #24 and points raised in your PRs, it looks like you are constantly recreating the `layout_index.sqlite`

### CODE REVIEW

File: abide/README.md [hunk 1]
```diff
@@ -21,54 +21,59 @@ A BIDS competable version of the ABIDE dataset are avalible through [datalad rep
...
-ARCHIVE="/nearline/ctb-pbellec/giga_preprocessing_2/abide1_fmriprep-20.2.7lts/abide1_fmriprep-20.2.7lts"
-SCRATCH_TARGET="~/scratch/abide1_fmriprep-20.2.7lts"
+If you are rerunning the connectome (QC not performed)
 
-cd $ARCHIVE
-SITES=`ls`
-mkdir $SCRATCH_TARGET
-cd $SCRATCH_TARGET
+1. Run one denoising strategy on one atlas, and use the `--reindex-bids` flag.
+2. Run one denoising strategy on the remaining atlas. Use the same path binding as the input (fMRIPrep derivative directory).
```
Comment (clarkenj) [inline abide/README.md:65]: I'm a bit confused here. After step 1, do I run all remaining atlases and strategies in parallel? Or do I need to run one strategy for each atlas first?

Comment (htwangtw) [reply abide/README.md:65]: one strategy for each atlas first.

This is for preventing the nodes locking files if they are saving to the same path (for example multiple jobs creating the grey matter mask and save to the same location)

Comment (clarkenj) [reply abide/README.md:65]: Ok,  with the `--reindex-bids` flag?

**Comment by htwangtw** [comment]
> -you mean with the reindex-bids flag? I also tried without it. Will follow new instructions from scratch and see.

The purpose of `--reindex-bids` is to recreate the `layout_index.sqlite`. 
If you have a locked / incomplete `layout_index.sqlite`, running without the flag will run into errors depending on the state of the `layout_index.sqlite`

File: abide/README.md (hunk 1 above)
Comment (htwangtw) [reply abide/README.md:65]: We don't want to recreating the `layout_index.sqlite` in step two.

**Pull request merged**